modeling-cmds/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "kittycad-modeling-cmds"
-version = "0.1.17"
+version = "0.2.0"
```

```suggestion
version = "0.1.18"
```
Just use 0.1.18 and ignore the semver check linter